StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -95,13 +95,12 @@ class NativeFlowController {
             self?.closeAuthFlow()
         }
         if showConfirmationAlert {
-            CloseConfirmationAlertHandler.present(
-                businessName: dataManager.manifest.businessName,
-                showNetworkingLanguageInConfirmationAlert: (dataManager.manifest.isNetworkingUserFlow == true && navigationController.topViewController is NetworkingLinkSignupViewController),
-                didSelectOK: {
+            let closeConfirmationViewController = CloseConfirmationViewController(
```

Comment:
would `closeConfirmationViewController` be cleaned up if not inlined here? Or this is a swift custom?

Reply:
hmm its debatable

I usually do like to inline things, but I do think its fair to say that its more common to separate the creation and presentation

for example, if you ask ChatGPT to generate code, it generates a very generic/common structure:

```
    func presentNewViewController() {
        let newViewController = NewViewController() // Create the new view controller
        newViewController.modalPresentationStyle = .fullScreen // Optional: Set the presentation style
        self.present(newViewController, animated: true, completion: nil) // Present the new view controller
    }
```